include/lsst/daf/base/PropertySet.h

```diff
@@ -172,6 +173,15 @@ class PropertySet : public Persistable, public Citizen {
      */
     std::type_info const& typeOf(std::string const& name) const;
 
+    /**
+     * Get type info for the specified class
+     *
+     * Implemented in the .cc file to work around symbol visiblity issues on macOS
+     * e.g. https://github.com/pybind/pybind11/issues/1503
+     */
```

Comment:
The location of the implementation doesn't seem like something we want in the API documentation. Consider moving to a normal comment.

Reply:
Should this be a method on `PropertySet` in the first place? If it's only needed for pybind11, maybe it should be a separate function (in the same files as now, I take it?).

Reply:
It is also used by C++ code. I'll change the "Implemented" comment as per your request.